test/evergreen.yml

```diff
@@ -385,6 +386,18 @@ functions:
       permissions: public-read
       content_type: text/plain
       remote_file: wiredtiger/${build_variant}/${revision}/artifacts/${task_name}_${build_id}/
+  "upload test/format config":
+    command: s3.put
+    type: setup
+    params:
+      aws_secret: ${aws_secret}
+      aws_key: ${aws_key}
+      local_files_include_filter:
+        - wiredtiger/cmake_build/test/format/RUNDIR*/CONFIG
```

Comment:
@etienneptl have we tested what happens when there's multiple RUNDIR directories?
i.e. if we're running 6 tests in parallel and only one fails, will we only upload the failing CONFIG, or could we detect that passing configs as well?

Reply:
Please see the JIRA ticket for a patch build where this is tested. The other runs are killed and only the failing one is left behind. If there are N failures at the same, time, I am not sure what happens in this case though.

Reply:
Though I'm interested to know how it behaves with multiple format failures in a single Evergreen task (e.g. those 6 hrs long ones), if it's hard to test I'm fine with the current solution, as for each format failure instance the CONFIG will be dumped in the task log. I assume this new upload step is to provide better accessibility (without users clicking into the task log and scrolling to the CONFIG part).

Reply:
I agree with you, @lukech and I am pretty sure that if we have N failures, we will just have one in Evergreen. But it's better than what we have now.
One idea we discussed with @ajmorton would be to give unique names for each failing config so they would appear individually in Evergreen.

Reply:
I can answer this question. If we have multiple RUNDIR failures, it will upload all CONFIGs since the local file name is CONFIG, it will most likely overwrite the existing CONFIG file that has already been uploaded.

Reply:
> give unique names for each failing config so they would appear individually in Evergreen.

That's a good idea. Matching each one with the specific test may need some manual effort, but it's a step forward compared to what we have now.

Reply:
I would recommend adding a `preserve_path: true`, which should prefix the path of where the file is located, so it doesn't overwrite with each other. 

Reference: https://docs.devprod.prod.corp.mongodb.com/evergreen/Project-Configuration/Project-Commands#s3put-with-multiple-files

Reply:
Thank you, @jiechenbo, I was not aware of that setting, added in 59063c58776eadb31b15016c89731dde243cb2cd.

Reply:
I think this only works if you give different paths to `local_files_include_filter`.